tests/Doctrine/Tests/ODM/PHPCR/Functional/Translation/AttributeTranslationStrategyTest.php

```diff
@@ -118,6 +118,74 @@ public function testLoadTranslation()
         $this->assertEquals(array(), $doc->getSettings());
     }
 
+    public function testSubRegionSaveTranslation()
+    {
+        // First save some translations
+        $data = array();
+        $data['topic'] = 'Some interesting subject';
+        $data['text'] ='Lorem ipsum...';
+
+        $node = $this->getTestNode();
+
+        $strategy = new AttributeTranslationStrategy($this->dm);
+        $strategy->saveTranslation($data, $node, $this->metadata, 'en');
+
+        // Save translation in another language
+
+        $data['topic'] = 'Some interesting american subject';
+
+        $strategy->saveTranslation($data, $node, $this->metadata, 'en_US');
+        $this->dm->flush();
+
+        // Then test we have what we expect in the content repository
+        $node = $this->session->getNode('/' . $this->testNodeName);
+
+        $this->assertTrue($node->hasProperty(self::propertyNameForLocale('en', 'topic')));
+        $this->assertTrue($node->hasProperty(self::propertyNameForLocale('en_US', 'topic')));
```

Comment:
should we make it an optional dependency but check to fail early when multilanguage is configured to say you need it? when not having multilang at all, its unneeded.